/**:
  ros__parameters:
    devices: ["*"]
    monitor_program: "greengrass"
    enable_traffic_monitor: true
    crc_error_check_duration: 1
    crc_error_count_threshold: 1
    reassembles_failed_check_duration: 1
    reassembles_failed_check_count: 1
    udp_buf_errors_check_duration: 1
    udp_buf_errors_check_count: 1
